### PR TITLE
Remove docs/google.md

### PR DESCRIPTION
**Pull request by wrp**
The current document references a broken link

### CODE REVIEW

**Comment by DirectXMan12** [comment]
cc @piosz @kawych

File: docs/google.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Heapster metrics are now being exported to Google Cloud Monitoring as custom met
 ### Metrics Dashboard
 To access the Google Cloud Monitoring dashboard go to: [https://app.google.stackdriver.com/](https://app.google.stackdriver.com/). Create a new dashboard and add the desired charts. Select the *Custom Metric* Resource Type and all Heapster metrics are under the `kubernetes.io` namespace. You can narrow down the query by the metric labels provided.
 
-It is also possible to query the Google Cloud Monitoring data directly using their [custom metric read API](https://cloud.google.com/monitoring/v2beta2/timeseries/list).
+It is also possible to query the Google Cloud Monitoring data directly using their [custom metric read API](https://cloud.google.com/monitoring/custom-metrics).
```
Comment (kawych) [inline docs/google.md]: Please point to the specific section with reference to API documentation: https://cloud.google.com/monitoring/custom-metrics/reading-metrics

**Comment by kawych** [comment]
Please note that this documentation is outdated, see #1715

**Comment by andyxning** [comment]
Seems we need to delete the google.md file instead of patch it.

**Comment by kawych** [comment]
@wrp 
As @andyxning mentioned, can you change this PR to remove google.md file instead of changing this link, or just close this PR (then I'll file a PR to remove the outdated docs)

**Comment by kawych** [comment]
/lgtm
Thanks! Please squash the commits.

**Comment by wrp** [comment]
Squashed and rebased

**Pull request merged**